StRoot/StBFChain/BigFullChain.h

```diff
@@ -1734,6 +1737,10 @@ Bfc_st BFC[] = { // standard chains
                                                                              "sTGC Point maker",    kFALSE},
   {"FttQA","","fttChain","","StFttQAMaker","StFttQAMaker", "sTGC Raw hit QA maker",                 kFALSE},
 
+  // forward track maker
+  {"FwdTrack","","","","StFwdTrackMaker",
+  "XMLIO,genfit2,KiTrack,StarGeneratorUtil,libMathMore,StEventUtilities,StEpdUtil,StFwdTrackMaker", 
+                                                            "Forward Track Maker",                  kFALSE},
```

Comment:
I may be missing something but I don't see where StFwdTrackMaker depends on either libMathMore or StEpdUtil libraries. This needs to be confirmed

Also, what is that XMLIO library? I think the library name is libxml2